docs/tutorials/t2hahn_characterization.ipynb

```diff
@@ -0,0 +1,590 @@
+      "ibmqfactory.load_account:WARNING:2022-01-05 23:14:31,530: Credentials are already in use. The existing account in the session will be replaced.\n",
+      "c:\\users\\014780756\\documents\\github_2\\qiskit_experiments\\curve_analysis\\guess.py:188: RuntimeWarning: invalid value encountered in log\n",
+      "  coeffs = np.polyfit(x, np.log(y), deg=1)\n",
+      "c:\\users\\014780756\\documents\\github_2\\qiskit_experiments\\curve_analysis\\curve_analysis.py:865: UserWarning: All initial guesses and parameter boundaries failed to fit the data. Please provide better initial guesses or fit parameter boundaries.\n",
+      "  warnings.warn(\n"
+     ]
+    }
+   ],
+   "source": [
+    "from qiskit import IBMQ\n",
+    "from qiskit_experiments.library.characterization.t2hahn import T2Hahn\n",
+    "import qiskit\n",
+    "from qiskit_experiments.library import T2Ramsey\n",
+    "import numpy as np\n",
+    "\n",
+    "def backend_fetcher():\n",
+    "    # TOKEN = \"\"\n",
+    "    # IBMQ.save_account(TOKEN)\n",
+    "    IBMQ.load_account() # Load account from disk\n",
+    "    provider = IBMQ.get_provider(hub='ibm-q')\n",
+    "    backend = provider.get_backend('ibmq_manila')\n",
+    "    backend_properties = backend.properties()\n",
+    "    return backend, backend_properties\n",
+    "\n",
+    "\n",
+    "backend, backend_properties = backend_fetcher()\n",
+    "estimated_t2hahn = backend_properties.t2(0)\n",
+    "\n",
+    "# Hahn Echo experiment parameters\n",
+    "qubit_hahn = 0\n",
+    "conversion_factor = 1e-6\n",
+    "delays2 = np.append(\n",
+    "                    (np.linspace(1.0, 10.0, num=37)).astype(float),\n",
+    "                    (np.linspace(10.5, 45.0, num=70)).astype(float),\n",
+    "                )\n",
+    "delays2 = [float(_) * conversion_factor for _ in delays2]\n",
+    "num_echoes = 4\n",
+    "\n",
+    "exp3_hahn = T2Hahn(qubit_hahn, delays2, num_echoes=num_echoes, backend=backend)\n",
+    "exp3_hahn.analysis.set_options(p0={\"amp\": 0.5, \"tau\": estimated_t2hahn, \"base\": 0.5}, plot=True)\n",
+    "expdata_hahn = exp3_hahn.run(backend=backend, shots=2000).block_for_results()\n",
+    "\n",
+    "# Ramsey experiment parameters\n",
+    "qubit_ramsey = 0\n",
+    "delays_ramsey = list(range(1, 350, 2))\n",
+    "delays_ramsey = [float(_) * conversion_factor for _ in delays_ramsey]\n",
+    "\n",
+    "# Create a T2Ramsey experiment. Print the first circuit as an example\n",
+    "backend, backend_properties = backend_fetcher()\n",
+    "exp_ramsey = T2Ramsey(qubit_ramsey, delays_ramsey, osc_freq=backend_properties.frequency(0), backend=backend)\n",
+    "\n",
+    "# Analysis\n",
+    "default_p0 = {\n",
+    "            \"A\": 0.5,\n",
+    "            \"T2star\": estimated_t2hahn,\n",
+    "            \"f\": backend_properties.frequency(0),\n",
```

Comment:
What you need here is the estimate for the frequency error, which is not the same as the qubit frequency

Reply:
Changed the guess for frequency to be 2% of the qubit frequency:
```
freq_diff = backend_properties.frequency(0) * 0.02
exp_ramsey = T2Ramsey(qubit_ramsey, delays_ramsey, osc_freq=freq_diff, backend=backend)
```

Reply:
This section was removed from the tutorial.